src/Spanner/Database.php

```diff
@@ -511,6 +516,10 @@ public function iam()
      */
     public function snapshot(array $options = [])
```

Comment:
We need to document `@throws` here and on `transaction` for the `BadMethodCallException`